docs/fieldnorm/control_of_documents.md

```diff
@@ -1,25 +1,65 @@
+
+## Why do Bioinformaticians need a different QMS
+
+Bioinformaticians create workflows collaboratively on a computer within a git environment. Git is a distributed version control system that tracks changes in any set of computer files. This can be used to track the changes of the code itself, but also in its documentation, which can describe the Standard Operating Procedures to use it. Git comes in with a set of features allowing to track issues and their resolution, new release of a workflow and its newest features. As the workflow often get upgraded and updated, their documentation, as well, is susceptible to change. Which is why many Bioinforaticians have turned their back from a standard QMS to have their own documentation system, often linked to git. Such system also offer possibilities to copy-paste code blocks while performing the SOP for a given flow, making it so much easier and less prone to errors. Examples of those documentations systems are:
 
 1. Github pages
-2. Gitlab
-3. Confluence
\ No newline at end of file
+2. Gitlab pages
+3. Confluence
+
+Many of those tools, if used in a specific way, can allow Bioinformaticians to work in a comfortable environement, while complying to the ISO norms requirements. They are of course multiple ways to implement such a system, and as long as it is documented how your system complies to the ISO requirements, all is well. If you are looking for some inspirations, in the next paragraph we will describe an example on how to use one this tool to follow the ISO requirements for both controlling the workflows/software you develop, and their associated documentation/SOP.
+
+## Example case : The usage of GitLab
+
+### Central gathering of all SOP on Gitlab pages
+_I wanted to add images to illustrates but I did not found out how_ :(
+
+Using gitlab pages and mkdocs, you can easily create a beautiful hub to store all your SOP. If using the following guidelines, then you will comply with each of the ISO requirements mentionned above : 
+
+1. Each SOP will become a markdown file. It's name will include a number for easy and unique identification of the document. The numbering can also be used to classify each SOP into categories. Example :
+- 10 - Data Access
+    - 11 - GenomeScan -> 11_genomescan.md
+    - 12 - InternalSequencers -> 12_internalsequencers.md
+- 20 - Data Processing
+    - 21 - WES -> 21_wes_flow.md
+    - 22 - RNAseq -> 22_rnaseq_analysis.md
+
+2. The repository is set such only restricted people who manages the documentation approves new entries to be added into the main document (dev -> main). And rules are set so each new feature or entry is approved by the competent person before being accepted (feature -> dev)
+
+3. The repository is set such as anyone who see something out to date can work on updating the document and make a request to the documentation managers so they can accept the update.  Besides, at the footer of each page, you can see the date of last modification.
```

Comment:
I'm not sure whether this suffices as having "periodic review".

Reply:
I think it definitely does not 😛 . But I could not find a system to set reminders to update a certain page, in gitlab at least. This part is openn to suggestion !